test/runtests.jl

```diff
@@ -81,25 +82,27 @@ closures = (
     :VerstappenAnisotropicMinimumDissipation
 )
 
-@testset "Oceananigans" begin
-    include("test_grids.jl")
-    include("test_fields.jl")
-    include("test_halo_regions.jl")
-    include("test_operators.jl")
-    include("test_solvers.jl")
-    include("test_coriolis.jl")
-    include("test_surface_waves.jl")
-    include("test_buoyancy.jl")
-    include("test_models.jl")
-    include("test_time_stepping.jl")
-    include("test_boundary_conditions.jl")
-    include("test_forcings.jl")
-    include("test_turbulence_closures.jl")
-    include("test_dynamics.jl")
-    include("test_diagnostics.jl")
-    include("test_output_writers.jl")
-    include("test_regression.jl")
-    include("test_examples.jl")
-    include("test_abstract_operations.jl")
-    include("test_verification.jl")
+with_logger(ModelLogger()) do
```

Comment:
This is pretty clean! I guess it makes sense to use `with_logger` here but for log messages peppered around the rest of the package would `global_logger(ModelLogger())` be better?

Reply:
I think so! Do you think the line could be added in `Oceananigans.jl`?

Reply:
Yeah we should do that, hopefully it doesn't also effect other packages. But we can leave it for another PR to avoid dragging this one out.